gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -0,0 +1,25 @@
+openapi: "3.0.2"
+info:
+  title: Zeebe REST API
+  version: "0.1"
+  description: API for communicating with the Zeebe cluster.
+  license:
+    name: Zeebe Community License Version 1.1
+    url: https://github.com/camunda/zeebe/blob/main/licenses/ZEEBE-COMMUNITY-LICENSE-1.1.txt
+  externalDocs:
+    description: Find out more
+    url: https://docs.camunda.io/docs/apis-tools/zeebe-api-rest/overview/
```

:x: This is invalid, it's a root property, and not a member of `info`

```suggestion
externalDocs:
  description: Find out more
  url: https://docs.camunda.io/docs/apis-tools/zeebe-api-rest/overview/
```